Week1/Section2/Exercise9/Source9.cpp

```diff
@@ -0,0 +1,22 @@
+/*
+* Exercise 9 - Lowercase Vowel
+Write a program that will receive an alphabetical character from the user. Then, tell them if the character is a lowercase vowel. Make sure to do this using a single if statement.
+
+Example output
+Hello! Please enter an alphabetical character: c
+Your character is not a lowercase vowel.
+*/
+
+#include <iostream>
+
+int main() {
+	char letter;
+	std::cout << "Hello! Please enter an alphabetical character: ";
+	std::cin >> letter;
+
+	if (islower(letter) && (letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u')) {
```

Comment:
There's no need to check for islower here, you are already checking against lowercase characters :)

Reply:
😮 wow you just blow my mind jajaja 😅